etc/clang-format-all.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+. "$(dirname "${BASH_SOURCE[0]}")/../.evergreen/init.sh"
+
+files="$(find "$LIBMONGOCRYPT_DIR/src" -type f -name '*.c' -o -name '*.h' -o -name '*.cpp' -o -name '*.hpp')"
+readarray -t files <<< "$files"
+bash "$LIBMONGOCRYPT_DIR/etc/clang-format.sh" \
+    --style="file:$LIBMONGOCRYPT_DIR/.clang-format" \
```

Comment:
It seems like clang-format doesn't support an option to guarantee the presence of an EOF newline. Is this something we can add to either `clang-format-all.sh` or `clang-format.sh` to ensure [C standard conformance](https://port70.net/~nsz/c/c99/n1256.html#5.1.1.2p1)?

> A source file that is not empty shall end in a new-line character, which shall not be immediately preceded by a backslash character before any such splicing takes place.

Reply:
It's possible, yes, but may require additional scripting. (That seems like a significant omission for clang-format.)

Reply:
TIL. Unfortunately, version 16 is still in an RC stage, so there is no PyPI package yet. It should be available in March. We can defer that changeset for a month, then bump to the new version once it's available.